Modules/System/Email/src/Email/Outbox/EmailOutbox.Table.al

```diff
@@ -109,4 +109,54 @@ table 8888 "Email Outbox"
         {
         }
     }
+
+    procedure GetMessageId(var MessageId: Guid)
+    begin
+        MessageId := Rec."Account Id";
+    end;
+
+    procedure GetAccountId(var AccountId: Guid)
+    begin
+        AccountId := Rec."Account Id";
+    end;
+
+    procedure GetUserSecurityId(var UserSecurityId: Guid)
+    begin
+        UserSecurityId := Rec."User Security Id";
+    end;
+
+    procedure GetConnector(): Enum "Email Connector"
+    begin
+        exit(Rec.Connector);
+    end;
+
+    procedure GetEmailStatus(): Enum "Email Status"
+    begin
+        exit(Rec.Status);
+    end;
+
+    procedure GetDescription(): Text[2048]
```

Comment:
I'd suggest dropping the size of the result to avoid breaking changes if we decide to change it.